gradient/api_sdk/s3_uploader.py

```diff
@@ -48,9 +48,15 @@ def _upload(self, url, data):
         :param str url:
         :param encoder.MultipartEncoderMonitor data:
         """
+
+        data_content_type = data.content_type or ""
+
+        if "multipart/form-data" in data.content_type:
+            data_content_type = "application/zip"
```

Comment:
can we just change the header like this, or do we have to actually convert the data to this type?  cc @bbatha

Reply:
on second thought, this if statement should not be necessary. we pass in a "contentType" from ps-api so that value should be in the `client.headers` and not `data.content_type`

Reply:
i know for modelUpload, when you upload a file from ps-web, the browser filepicker determines the file's `contentType` and passes that value to the ps-api endpoint (each uploaded file has a type: https://github.com/Paperspace/PS_WEB/blob/master/client/scripts/react/containers/console/actions/modelsActions.js#L124)

for CLI however, i think this is the place we'd need to determine `contentType` and pass it to the ps-api endpoint

Reply:
i found this `https://github.com/Paperspace/gradient-cli/blob/9ee76792f69294398f9fdd894b7b59534794f141/gradient/api_sdk/s3_uploader.py#L95-L97` which is used for model upload and experiment upload